### PR TITLE
New: Various Healthcheck enhancements | Fixed: Various bad Healthcheck links

### PR DESCRIPTION
**Pull request by bakerboy448**
New: Localized HealthChecks
Fixed: Healthcheck Wiki Links
Various pulls from Radarr;  Completed at the file - not commit - level

#### Database Migration
NO

#### Description
- Brings over numerous Radarr commits for healthchecks including correct strings and previously missed localizations
- Builds
- Tests (almost) all work

#### Todos
- [x] Tests **Need some help fixing the broken ImportList healthcheck; I'm not seeing it**
- [X] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json) - Added some; likely missing some though. also noticed unused strings exist @austinwbest cleanup in aisle translate json? :D
- [ ] [Wiki Updates](https://wiki.servarr.com) - @RobinDadswell we did all the wikis right?

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Core.Test/HealthCheck/Checks/AppDataLocationFixture.cs [hunk 1]
```diff
@@ -9,6 +11,14 @@ namespace NzbDrone.Core.Test.HealthCheck.Checks
...
     public class AppDataLocationFixture : CoreTest<AppDataLocationCheck>
     {
+        [SetUp]
+        public void Setup()
+        {
+            Mocker.GetMock<ILocalizationService>()
+                  .Setup(s => s.GetLocalizedString(It.IsAny<string>()))
+                  .Returns("Some Warning Message");
+        }
+
```
Comment (RetroDadson) [inline src/NzbDrone.Core.Test/HealthCheck/Checks/AppDataLocationFixture.cs:21]: Add this to the fixture of the failing test and it should start working again for you

**Review by RetroDadson — DISMISSED** [review]

**Comment by ta264** [comment]
> Various pulls from Radarr; Completed at the file - not commit - level

@bakerboy448 if you're going to do that, can you add in the PR `- Fixed #xxx` for each pull issue you've covered with this?

**Comment by bakerboy448** [comment]
@ta264 there are no Radarr Pull issues to tag.

By radarr pulls I mean various (upwards of a dozen I think) radarr commits that we're never commoned and thus never pulled nor have an issue.

I'll de-conflict shortly

**Review by ta264 — CHANGES_REQUESTED** [review]
Looks like a bunch of the translations you reference are missing.  eg IndexerStatusCheckAllClientMessage

**Review by ta264 — APPROVED** [review]

**Pull request merged**